components/feature/pwa/src/test/java/mozilla/components/feature/pwa/ext/ActivityKtTest.kt

```diff
@@ -53,4 +63,16 @@ class ActivityKtTest {
             verify(activity).requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_USER
         }
     }
+
+    @Test
+    fun `WHEN reportFullyDrawnSafe is called THEN reportFullyDrawn is called`() {
+        activity.reportFullyDrawnSafe()
+        verify(activity).reportFullyDrawn()
+    }
+
+    @Test
+    fun `GIVEN reportFullyDrawn throws a SecurityException wHEN reportFullyDrawnSafe is called THEN the exception is caught`() {
+        `when`(activity.reportFullyDrawn()).thenThrow(SecurityException())
+        activity.reportFullyDrawnSafe() // If an exception is thrown, this test will fail.
```

Comment:
Perhaps this can be made more testable by passing a `Logger` instance instead of the tag? Fenix can then store a performance logger instead of a performance tag.

Reply:
While that's true, I don't think this code will change in the near future (low risk) and I have higher priority things to focus on (comparatively low reward) so I'd prefer to land this as is.

Reply:
But now I realize my log statement can be better so I'll make the change.